package.json

```diff
@@ -28,7 +28,7 @@
     "docs:build": "cd ./website && npm run build && cd ..",
     "docs:deploy": "./scripts/updateDocs.js",
     "test": "run-s test:*",
-    "test:eslint": "eslint --cache packages examples scripts tests",
+    "test:eslint": "./node_modules/.bin/eslint --cache packages examples scripts tests",
```

Comment:
not needed .. per default NPM always looks into this directory first to find the command